internal/commands/app/create.go

```diff
@@ -72,6 +73,7 @@ func (cmd *CommandCreate) Flags() []flags.Flag {
 				Name: flagClusterServiceName,
 				Usage: flags.Usage{
 					Description: "Specify the Realm app Service name to reference your Atlas cluster",
+					Note:        "Service names will be overwritten when creating a template app",
```

Comment:
[q] I didn't catch/think about this until now, but should we stop referring to services here? should this just be "Data Source names..." and so on?

Reply:
I feel like we can just change references to "service" here, I wouldn't consider that a breaking change... however, we probably _do_ need to leave the `cluster-service-name` flag/arg for now (to avoid a breaking change) and just add an additional `data-source` arg.

cc @makesitgo

Reply:
Agreed that changing the "note" text should be fine.  Do you mind elaborating on what the extra `data-source` arg would be?  Or are you suggesting replacing `cluster-service-name` with `cluster-data-source-name`?

Whatever we do with this one, there's an analogous `datalake-service-name` flag we should consider as well.  And if we're in the mood for breaking changes, maybe we remove the support for "multiple data sources at once", which would then eliminate the `*-service-name` flags all together (we could align with templates and go back to forcing `mongodb-atlas` as the name).

Reply:
hmm I could be wrong, but maybe Chris is suggesting making 2 flags do the same thing: `--cluster-service-name` and `--cluster-data-source-name`. I'm hoping there's a way to alias flags so we don't need to manually account for that in our code if we end up going this route.

That said, I think this is a good opportunity to audit the CLI, Frontend (and maybe backend) to find wherever we still refer to data sources or HTTP endpoints as services and change them accordingly.

I can make a new ticket for this if we want to do this

Reply:
https://jira.mongodb.org/browse/REALMC-9950

Reply:
Hmm, I'm not sure if this is a valid concern, but I think that depending on the solution we end up with in REALMC-9950, we may want to leave this flag alone.

I'd rather just keep the descriptions aligned with their flag name until we know for sure that the change is needed.

Reply:
no strong opinions on my end, I'd maybe just ask sumedha/drew if they want this quick update or not